components/time.scss

```diff
@@ -0,0 +1,4 @@
+$fd-fonts-path: "../scss/fonts/";
+
+@import "../scss/fonts/72";
```

Comment:
Not sure why I didn't see this before, but the font isn't actually needed for this component.  The form input will bring the font with it.